### PR TITLE
Fix for homepage toggle page shake

### PR DESCRIPTION
**Pull request by jasperfurniss**
#### Screens
Screen Recording 2020-12-18 at 12 31 49 PM

#### Breaking Changes
No

#### Runway Ticket URL
https://nitro.powerhrg.com/runway/backlog_items/NUXE-365

#### How to test this
Click the react/rails toggle and ensure on all screens that there is no shaking.

#### Checklist:

- [x] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [x] **URGENCY** Please select a release milestone
- [x] **DEPLOY** Please add the `Milano` label when you are ready for a review.
- [x] **SCREENSHOT** Please add a screen shot or two.
~- [] **SPECS** Please cover your changes with specs.~
- [x] **READ DOCS** Please make sure you have read and understand the [Playbook Release Process](https://github.com/powerhome/playbook/wiki/Playbook-Releases)

*The normal release cut off deadline is 3p EDT each week. Please reach out to the release team if you have an urgent request or need a release off cycle.*

### CODE REVIEW

File: playbook/app/pb_kits/playbook/packs/site_styles/_landing-page.scss [hunk 1]
```diff
@@ -9,6 +9,7 @@
     background-color: $bg_dark;
     border-radius: $border_rad_heaviest;
     box-shadow: $shadow_deepest;
+    min-width: 733px;
```
Comment (thestephenmarshall) [inline playbook/app/pb_kits/playbook/packs/site_styles/_landing-page.scss:12]: Curious on the specific number `733px` here. Could we possibly make use of [screen size tokens](https://github.com/powerhome/playbook/blob/master/playbook/app/pb_kits/playbook/tokens/_screen_sizes.scss#L5)?

Comment (jasperfurniss) [reply playbook/app/pb_kits/playbook/packs/site_styles/_landing-page.scss:12]: 733px is the smallest width that lined the black edge of the card up with the avatar card above it, without changing the original design.

Comment (jasperfurniss) [reply playbook/app/pb_kits/playbook/packs/site_styles/_landing-page.scss:12]: I can show you what it would look like with that token.

Comment (jasperfurniss) [reply playbook/app/pb_kits/playbook/packs/site_styles/_landing-page.scss:12]: https://share.getcloudapp.com/ApuGJ01k @thestephenmarshall

Comment (thestephenmarshall) [reply playbook/app/pb_kits/playbook/packs/site_styles/_landing-page.scss:12]: I see. Ok just figured I would ask!

**Review by thestephenmarshall — APPROVED** [review]

File: playbook/app/pb_kits/playbook/packs/site_styles/_landing-page.scss (hunk 1 above)
Comment (jasperfurniss) [reply playbook/app/pb_kits/playbook/packs/site_styles/_landing-page.scss:12]: Of course. I was hoping one would! I don't like not using them for sure. 100%.

**Review by jasoncypret — APPROVED** [review]

**Pull request merged**